src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -45,8 +45,8 @@ public static String getPersonDetails(Person person) {
         sb.append(PREFIX_MEDICAL_HISTORY + person.getMedicalHistory().value + " ");
         sb.append(PREFIX_LOCATION + person.getLocation().value + " ");
         person.getOneTimeSchedules().stream().forEach(
-                s -> sb.append(PREFIX_ONETIMESCHEDULE + s.getDate() + " " + s.getStartTime() + " "
-                        + s.getEndTime() + " ")
+                s -> sb.append(PREFIX_ONETIMESCHEDULE + s.getDate().toString()
```

Comment:
```suggestion
                s -> sb.append(PREFIX_ONETIMESCHEDULE + s.getDateString()
```
just want to check if LocalDate.toString() should be here? Since this will produce any given LocalDate in a String format of "yyyy-mm-dd"